addons/l10n_in/models/account_tax.py

```diff
@@ -119,3 +119,32 @@ def _l10n_in_get_hsn_summary_table(self, base_lines, display_uom):
             'display_uom': display_uom,
             'items': items,
         }
+
+
+class AccountTaxGroup(models.Model):
+    _inherit = 'account.tax.group'
+
+    l10n_in_is_tcs = fields.Boolean("Is TCS")
+    l10n_in_is_tds = fields.Boolean("Is TDS")
+    l10_in_is_threshold = fields.Boolean("TCS/TDS Threshold Limit")
+    l10n_in_consider_tax = fields.Selection([
+            ('untaxed_amount', 'Untaxed Amount'),
+            ('total_amount', 'Total Amount'),
+        ], string="Consider", default='untaxed_amount', required=True)
```

Comment:
Did you check this doesn't affect other localization?
`required=True` same for the other field below??